validation/genesis/genesis-allocs_test.go

```diff
@@ -184,6 +184,21 @@ func testGenesisAllocs(t *testing.T, chain *ChainConfig) {
 		delete(g.Alloc, common.HexToAddress("0x5c69bee701ef814a2b6a3edd4b1652cb9cc5aa6f"))
 	}
 
+	// Ink Sepolia
+	if chainId == uint64(763373) {
```

Comment:
@bitwiseguy not sure how you want to sequence merging this PR and the original one. One option is to merge _only_ this exception code now, and rebase the other PR on `main`?

Reply:
@geoknee 
@bitwiseguy mentioned this PR replaces https://github.com/ethereum-optimism/superchain-registry/pull/698 , and it has all the same commits in it, so I think we can proceed with this one